media-sound/guitarix2/guitarix2-0.35.2.ebuild

```diff
@@ -0,0 +1,81 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI="5"
+
+[[ "${PV}" = "9999" ]] && inherit git-r3
+
+PYTHON_COMPAT=( python{2_7,3_{3,4}} )
+PYTHON_REQ_USE="threads(+)"
+inherit python-any-r1 waf-utils
+
+DESCRIPTION="A simple Linux Guitar Amplifier for jack with one input and two outputs"
+HOMEPAGE="http://guitarix.sourceforge.net/"
+
+RESTRICT="mirror"
```

Comment:
Lots of proaudio packages have this for some unknown reason, but it makes no sense for free software packages.